showcase/app/components/shw/text/index.ts

```diff
@@ -58,18 +71,18 @@ export default class ShwTextIndexComponent extends Component {
    * Sets the "variant" (style) for the text
    * Accepted values: see AVAILABLE_VARIANTS
    *
-   * @type {string}
+   * @type {TextVariants}
    *
    * @param variant
+   * @return {TextVariants}
    */
-  get variant() {
-    let { variant } = this.args;
+  get variant(): TextVariants {
+    const { variant } = this.args;
 
-    if (variant) {
-      variant = variant.toLowerCase();
```

Comment:
I'm not sure this check is needed anymore. As we convert pages to typescript, if the variant prop is typed incorrectly we can just fix it.

Reply:
I just confirmed that no instance of Shw::Text passes and incorrect variant prop.